docs/developers-guide/files/json-schema-ast.md

```diff
@@ -0,0 +1,49 @@
+# Json Schema AST
```

Comment:
This kind of documentation should go into the code as Elm doc instead of a separate document in my opinion.

Reply:
Fixed now